lib/common/filters/retrypolicyfilter.js

```diff
@@ -208,7 +208,7 @@ RetryPolicyFilter._shouldRetryOnError = function (statusCode, requestOptions) {
     }
     
     // When absorbConditionalErrorsOnRetry is set (for append blob)
-    if (requestOptions && requestOptions.absorbConditionalErrorsOnRetry) {
+    if (requestOptions && requestOptions.absorbConditionalErrorsOnRetry !== false) {
```

Comment:
I don't think you should add this for `absorbConditionalErrorsOnRetry`, because it's a options field which can be defined when calling the uploading appendblob API. 

 https://github.com/Azure/azure-storage-node/blob/66d24e89eeedf77ab83356c69fe5a39ed03b73b9/lib/services/blob/blobservice.node.js#L195

Reply:
Thanks, I missed that option. Since the default is to not have a retry policy then it makes sense that the default is to not absorb errors.